eclair-core/src/main/resources/reference.conf

```diff
@@ -14,8 +14,8 @@ eclair {
     port = 8080
     password = "" // password for basic auth, must be non empty if json-rpc api is enabled
   }
-
-  watcher-type = "electrum" // other *experimental* values include "bitcoinj" or "electrum"
+  
+  watcher-type = "electrum" // values include "bitcoind", "bitcoinj" and "electrum". bitcoinj and electrum are experimental.
```

Comment:
This should be applied to the `master` branch (but I don't really see the problem with current comment)

Reply:
That's because the Android version uses electrum but the comment only mentions electrum and bitcoinj. I added bitcoind for Android specifically.